lib/src/pdfjs_global.dart

```diff
@@ -29,7 +29,7 @@ enum VERBOSITY_LEVELS {
 }
 
 class PDFJS {
```

Comment:
It'd be nice to make this mirror the PDF.js API, but ultimately not a deal breaker, particularly as it's a violation of Dart naming standards.

Reply:
I guess I could make it `PdfjsLib` but I'm fine either way. The beauty of not changing it is this release won't actually break any consumers as far as I've tested